test/parallel/test-dgram-bind-error-repeat.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const dgram = require('dgram');
+
+// Regression test for https://github.com/nodejs/node/issues/30209
+// No warning should be emitted when re-trying `.bind()` on UDP sockets
+// repeatedly.
+
+process.on('warning', common.mustNotCall());
+
+const reservePortSocket = dgram.createSocket('udp4');
+reservePortSocket.bind(() => {
+  const { port } = reservePortSocket.address();
+
+  const newSocket = dgram.createSocket('udp4');
+
+  let errors = 0;
+  newSocket.on('error', common.mustCall(() => {
+    if (++errors < 20) {
+      const cb = errors === 20 ? common.mustCall() : common.mustNotCall();
```

Comment:
`errors` cannot be 20 here because if it is the `if` branch is not taken as `++errors` is used in the codition.

Reply:
Right, now sure what I was thinking there … :smile: